util/secret/certificates.go

```diff
@@ -63,36 +63,37 @@ type Certificates []*Certificate
 
 // NewCertificatesForInitialControlPlane returns a list of certificates configured for a control plane node
 func NewCertificatesForInitialControlPlane(config *v1beta1.ClusterConfiguration) Certificates {
-	if config.CertificatesDir == "" {
-		config.CertificatesDir = DefaultCertificatesDir
+	certificatesDir := DefaultCertificatesDir
```

Comment:
linter is complying about this, so I'm changing even if not directly related to this PR